sections/collage.liquid

```diff
@@ -509,8 +509,9 @@
         {
           "type": "text",
           "id": "description",
-          "default": "Describe the video for customers using screen readers",
-          "label": "Video alt text"
+          "default": "Describe the video",
+          "label": "Video alt text",
+          "info": "Describe the video to make it accessible for customers using screen readers."
```

Comment:
I think we would need to create the translation in the `en.default.json` here  for the label and info🤔

Reply:
I think I did it here because I couldn't find it in the `en.default.json` 🤔 I'll check again!